app/app.test.js

```diff
@@ -0,0 +1,91 @@
+const request = require('supertest');
+const app = require('./app');
+const environment = process.env.NODE_ENV || 'development';
+const configuration = require('../knexfile')[environment];
+const database = require('knex')(configuration);
+
+describe('Server', () => {
+
+  beforeEach(async () => {
+    await database.seed.run();
+  });
+
+  describe('init', () => {
+    it('should return a 200 status', async () => {
+      const res = await request(app).get('/');
+      expect(res.status).toBe(200);
+    });
+  });
+
+  describe('GET /api/v1/projects', () => {
+    it('should return a 200 and all of the projects', async () => {
+      const expectedProjects = await database('projects').select();
+
+      const res = await request(app).get('/api/v1/projects');
+
+      const response = JSON.stringify(res.body)
```

Comment:
I see what you mean about the JSON.stringify(). This is good for now, let's try to find a solution this week.